tests/src/Context/DrupalContext.php

```diff
@@ -38,6 +39,47 @@ public function assertContentPageByTitle($type, $title) {
   }
 
   /**
+   * Resolves contextual links directly, without the need for javascript.
+   *
+   * @Then I click the contextual link :linkText in the :region region
```

Comment:
I would still prefer this to be named something like "Then I click the "Edit Menu" link of the collection menu" and also hardcode the region or map the regions with the menu names. The user does not know the theme region names to begin with.